randovania/data/json_data/prime1.txt

```diff
@@ -2171,6 +2237,7 @@ Asset id: 3835971118
       Any of the following:
           Space Jump Boots and Heated Rooms Damage ≥ 165
           Morph Ball and Spider Ball and Heat-Resisting Suit
+          Morph Ball Bomb and Morph Ball and Heated Rooms Damage ≥ 135 and Lava Damage ≥ 147
```

Comment:
Missing Bomb Jump trick

Reply:
This was done with single bomb jumps, no DBJ was used. Should single bomb jumps be considered a trick as well when done in a place the player would not expect to be required to use morph ball?

Reply:
Personally, calling the trick double bomb jump is bad imo, since it would have to be a dbj. if the trick is called "Bomb Jump", then it can cover bomb jumps that arent intended, whether single/double, as well as include higher level bomb jumps like uber/high/halfpipe at higher difficulties, rather than separate tricks.

Reply:
I think you make sort of a good point. Since they're both "vanilla" techniques in a sense, I'll merge them together, though I'll probably have future unintended bomb techs like BSJ's, 3BJ's, &c. have their own tricks to give the user a degree of flexibility in controlling the tricks which the randomizer uses.

Reply:
BSJ would be its own trick yes, as seen for Echoes and Corruption